src/master-playlist-controller.js

```diff
@@ -72,6 +73,12 @@ const parseCodecs = function(codecs) {
   return result;
 };
 
+export const mapLegacyAvcCodecs = (codecString) => {
```

Comment:
This doesn't really affect anything since this function doesn't use `this`, but I think it's safer to not use arrow function syntax for top level functions just because it creates special behavior